### ISSUE TITLE
how to improve fps，I use 7 k80

### ISSUE THREAD

**Issue by LU-K-Brant** [Qidian213/deep_sort_yolov3#85]

**Comment by xxcheng0708** [comment]
Did you have any solutions?

**Comment by Reddyforcode** [comment]
i use a video capture Async method and works really well in a 1050ti and even better in a 2080ti

**Comment by linspace100** [comment]
> i use a video capture Async method and works really well in a 1050ti and even better in a 2080ti

How did you do the Async method? Could you please help me..?

**Comment by Reddyforcode** [comment]
https://github.com/Reddyforcode/thermal_camera_mtcnn/blob/master/videocaptureasync.py
remove from the 9th line to 17th line for just the source that you want to have 
```self.src=0
```
for your local webcam 

use:

````
from videocaptureasync import VideoCaptureAsync

cap = VideoCaptureAsync()
cap.start

while (True):
    _, frame = cap.read()

    #when exit
    #cap.stop()
```

or just don't process all the frames that you receive

**Comment by linspace100** [comment]
@Reddyforcode Wow amazing. I did yours and got a amazing fps. 


But, my model doesn`t work well. 
For example, in the demo file, line 89 to 100.
`if writeVideo_flag:
            # save a frame
            out.write(frame)
            frame_index = frame_index + 1
            list_file.write(str(frame_index)+' ')
            if len(boxs) != 0:
                for i in range(0,len(boxs)):
                    list_file.write(str(boxs[i][0]) + ' '+str(boxs[i][1]) + ' '+str(boxs[i][2]) + ' '+str(boxs[i][3]) + ' ')
            list_file.write('\n')
            
        fps  = ( fps + (1./(time.time()-t1)) ) / 2
        print("fps= %f"%(fps))`

It always print former fps like 6~8. But actual video in window is much faster than before. Maybe x5 or x10 that I think.

Do you think it is possible to show exact fps?

**Comment by guojingsai** [comment]
@linspace100 
Hi:
I'm seeing the same issue,could you fix the fps print issue?

**Comment by linspace100** [comment]
@guojingsai nope.. some people are struggling to solve the issue but it can`t be solved. Maybe you`d better use multiprocessing not async method. Async method cannot detect all of the objects in the frame.

**Comment by HirataYurina** [comment]
ok
